### DISCUSSION TITLE
Add shortest route profile to pedestrian navigation

### DISCUSSION THREAD

**Discussion by ZutgKcjA** [organicmaps/organicmaps#5575] [Ideas]
Please add a way to select a profile that shows the absolute shortest possible route to the destination in pedestrian navigation. The current routing profile does not find the shortest possible route to the destination, which results in lost time, since walking speed is the same anywhere (in cities, with no elevation). 

I assume the current profile does this because it is programmed to prefer quiet streets with fewer cars or some other reason I am not aware of. 

Find below an example of a bad route which loses 10 minutes (!) because it is about 600 m too long.

![Screenshot_20230719_054727_Organic Maps](https://github.com/organicmaps/organicmaps/assets/80195255/778c4ac6-9df1-4708-b174-faa06526caed)
![Screenshot_20230719_054908_Firefox](https://github.com/organicmaps/organicmaps/assets/80195255/6eba80a3-828d-4781-a5f9-d057d8c6b90c)
![Screenshot_20230719_054843_Firefox](https://github.com/organicmaps/organicmaps/assets/80195255/554df082-deae-4aae-91d3-23f9539e450a)

**Comment by biodranik** [comment]
The current profile finds the _fastest_ path, not the _shortest_ one. It looks like a bug @vng WDYT?

**Comment by ZutgKcjA** [comment]
Of course I don't know what the routing algorithm looks like in detail. But as far as pedestrian navigation is concerned, the shortest route is always going to be the fastest one (in built-up areas, extreme elevation notwithstanding), since walking speed is always going to be the same on all possible paths to the destination.

**Comment by vng** [comment]
Probably, there is a data issue in production now. I've checked on the latest master with the newly generated data (not in production) and got this (with approx starting point):
<img width="1123" alt="Screenshot 2023-07-23 at 09 31 36" src="https://github.com/organicmaps/organicmaps/assets/175612/5f894fe1-8052-48f9-9d85-979f9986287a">

**Comment by vng** [comment]
If after the app update (with next release) you will still have a bug, please send us the exact coordinates on support@organicmaps.app

**Comment by ZutgKcjA** [comment]
Version 2023.08.18-8-Google is again taking the "southern" route to the destination, adding about 600 m as compared to the optimal route. 

There were a couple of days in July (don't remember when I checked) where I can confirm that the profile took the northern route to the destination, but it still did not take the most optimal route, because it decided to go too far north. 

The screenshot posted by @vng shows the route going north of "Sonnenpark" along the "Sonnenstraße" and past the Shell station, while it would have been more optimal to take the road that goes almost parallel to the south, which is the L85 (Oberbilker Allee). 
The most glaring difference I can see between the L85 and Sonnenstraße is the fact that L85 is a major thoroughfare into the city, featuring 2-3 lanes in each direction for cars and a separated tram line in the middle, while Sonnenstraße is a very quiet residential street. 

Now, I understand this may be a desirable behavior for some users but most of the time for me, I just want to find the absolute shortest route, which is the fastest route for city navigation or perhaps even most cases. 
This discussion is of course complicated by the fact that as of the new version, the routing profile is navigating to the south as described in my original comment. 

This is why I reaffirm my conviction that we need 1. more transparancy as to how or why the particular route has been chosen and 2. a new routing profile for pedestrian navigation (in addition to the current profile) which calculates the absolute shortest route while disregarding how "loud" or busy a particular road along the route is. 

My exact coordinates for the start and end points are:
Start: 51.197735, 6.825146
End: 51.215346, 6.754307
